assets/js/components/ChartSourceController/components/BarChart/services/createChartJsConfig/index.js

```diff
@@ -10,214 +10,248 @@ import isObjectLiteral from './services/isObjectLiteral/index.js';
+                    },
+                }],
+                xAxes: [{
+                    barPercentage: 1,
+                    stacked: true,
+                    categoryPercentage: 0.6,
+                    ticks: {
+                        beginAtZero: true,
+                        maxRotation: rotateLabels ? 90 : 0,
+                        minRotation: rotateLabels ? 90 : 0,
+                        callback: value => (rotated ? value : `R${trimValues(value)}`),
+                    },
+                    gridLines: {
+                        color: 'transparent',
+                        display: true,
+                        drawBorder: false,
+                        zeroLineColor: '#ccc',
+                        zeroLineWidth: 1,
+                    },
+                }],
+            },
         },
-      },
-      legend: {
-        display: false,
-      },
-      scales: {
-        yAxes: [{
-          barPercentage: 0.8,
-          categoryPercentage: 1.0,
-          display: true,
-          gridLines: {
-            color: 'transparent',
-            display: true,
-            drawBorder: false,
-            zeroLineColor: '#ccc',
-            zeroLineWidth: 1,
-          },
-          ticks: {
-            display: rotated,
-            beginAtZero: true,
-            callback: value => (rotated ? `R${trimValues(value)}` : value),
-          },
-        }],
-        xAxes: [{
-          barPercentage: 1,
-          categoryPercentage: 0.6,
-          ticks: {
-            beginAtZero: true,
-            maxRotation: rotateLabels ? 90 : 0,
-            minRotation: rotateLabels ? 90 : 0,
-            callback: value => (rotated ? value : `R${trimValues(value)}`),
-          },
-          gridLines: {
-            color: 'transparent',
-            display: true,
-            drawBorder: false,
-            zeroLineColor: '#ccc',
-            zeroLineWidth: 1,
-          },
-        }],
-      },
-    },
-    plugins: [
-      {
-        afterDatasetsDraw: rotated || dynamicLabelPlugin,
-      },
-    ],
-  };
+        plugins: [
+            {
+                afterDatasetsDraw: rotated ? function (chart, options) {
+                    let ctx = chart.chart.ctx;
+
+                    ctx.textAlign = 'center';
+                    ctx.fillStyle = '#fff';
+                    ctx.font = "10px \"Helvetica Neue\", Helvetica, Arial, sans-serif";
+
+                    datasets.forEach(function (dataset, i) {
+                        if (dataset.stack !== 'Actual Expenditure') {
+                            return
+                        }
+                        let meta = chart.controller.getDatasetMeta(i);
+                        meta.data.forEach(function (bar, index) {
+                            const y = bar._model.y + ((bar._model.base - bar._model.y) / 2) + 5;
+                            ctx.fillText(`Q${i - 3}`, bar._model.x, y);
+                        });
+                    });
```

Comment:
a better way than Q${1-3} might have been if you could have a `quarter` property in `dataset`

Reply:
will do - thanks